src/smart-components/group/policy/add-policy/edit-policy-modal.js

```diff
@@ -0,0 +1,177 @@
+  updatePolicy
+}) => {
+  const [ selectedPolicy, setSelectedPolicy ] = useState({});
+  const [ inputValue, setInputValue ] = useState('');
+  const [ selectedUsers, setSelectedUsers ] = useState([]);
+  const [ optionIdx, setOptionIdx ] = useState(0);
+
+  const createOption = (label) => {
+    let idx = optionIdx;
+    setOptionIdx(optionIdx + 1);
+    return {
+      label,
+      value: `${label}_${idx}`
+    };
+  };
+
+  const setPolicyData = (policyData) => {
+    setSelectedPolicy(policyData);
+    if (policyData) {
+      setSelectedUsers(policyData.principals.map(user => (createOption(user.username))));
+    }
+  };
+
+  const fetchData = () => {
+    fetchPolicy(id).payload.then((data) => setPolicyData(data)).catch(() => setPolicyData(undefined));
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const onSubmit = data => {
+    const user_data = { ...data, user_list: selectedUsers.map(user => ({ username: user.label })) };
+    updatePolicy(user_data).then(() => fetchGroupPolicies()).then(push('/policies'));
+  };
+
+  const onCancel = () => {
+    addNotification({
+      variant: 'warning',
+      title: selectedPolicy ? 'Editing policy' : 'Adding policy',
+      description: selectedPolicy ? 'Edit policy was cancelled by the user.' : 'Adding policy was cancelled by the user.'
+    });
+    push('/policies');
+  };
+
+  const schema = {
+    type: 'object',
+    properties: {
+      name: { title: selectedPolicy ? 'Policy Name' : 'New Policy Name', type: 'string' },
+      description: { title: 'Description', type: 'string' }
+    },
+    required: [ 'name' ]
+  };
+
+  const handleChange = (value) => {
+    setSelectedUsers(value);
+  };
+
+  const handleInputChange = (val) => {
+    setInputValue(val);
+  };
+
+  const handleKeyDown = (event) => {
+    if (!inputValue) {return;}
+
+    switch (event.key) {
+      case 'Enter':
+      case 'Tab':
+        if (selectedUsers) {
+          if (!selectedUsers.find(user => (user.label === inputValue))) {
+            setSelectedUsers([ ...selectedUsers, createOption(inputValue) ]);
+          }
+        }
+        else {
+          setSelectedUsers([ createOption(inputValue) ]);
+        }
+
+        setInputValue('');
+        event.preventDefault();
+    }
+  };
+
+  return (
+    <Modal
+      isLarge
+      title={ selectedPolicy ? 'Edit policy' : 'Add policy' }
+      isOpen
+      onClose={ onCancel }
+    >
+      <Grid gutter="md" style={ { minWidth: '800px' } }>
+        <GridItem sm={ 6 }>
+          <FormRenderer
+            schema={ schema }
+            schemaType="mozilla"
+            onSubmit={ onSubmit }
+            onCancel={ onCancel }
+            formContainer="modal"
+            initialValues={ { ...selectedPolicy } }
+          />
+        </GridItem>
+        <GridItem sm={ 6 }>
+          <TextContent>
+            <Text component={ TextVariants.h6 }>Select Members for this policy.</Text>
+          </TextContent>
+          <CreatableSelect
```

Comment:
Create-able select is now part of data driven forms. You can use `selectVariant="createable"` prop to enable it. If you want to keep it separate from form but want have PF4 styling you can import just the select component from:
```JSX
import { rawComponents } from '@data-driven-forms/pf4-component-mapper'

<rawComponents.Select {...yourProps} />
```